api/query/cache/service/main.go

```diff
@@ -141,13 +141,19 @@ func searchHandler(w http.ResponseWriter, r *http.Request) {
 	// resident in `idx`. In the unlikely event that a run in `ids`/`runs` is no
 	// longer in `idx`, `idx.Bind()` below will return an error.
 	q := cq.PrepareUserQuery(ids, rq.AbstractQuery.BindToRuns(runs...))
+
+	// Configure format, from request params.
+	_, subtests := r.URL.Query()["subtests"]
+	opts := query.AggregationOpts{
+		IncludeSubtests: subtests,
+	}
```

Comment:
this is a boolean value?

Reply:
https://blog.golang.org/go-maps-in-action

> A two-value assignment tests for the existence of a key

`_` is the (ignored) value, `subtests` is whether the value was in the map.